build.gradle

```diff
@@ -11,7 +11,8 @@ apply plugin: 'maven-publish'
 sourceCompatibility = 1.6
 targetCompatibility = 1.6
 def releaseVersion = System.properties.RELEASE_VERSION
-version = releaseVersion ? releaseVersion : new SimpleDateFormat('yyyy-MM-dd\'T\'HH-mm-ss').format(new Date())
+//version = releaseVersion ? releaseVersion : new SimpleDateFormat('yyyy-MM-dd\'T\'HH-mm-ss').format(new Date())
+version = releaseVersion ? releaseVersion : '2.0.0-SNAPSHOT'
```

Comment:
This is used for deploying the newest build on bintray, by using a constant string we would always override the latest version, which is not what I want. The date is unique and clear for everybody who want's to use the latest (or one specific) build.